docs/spaces/index.asciidoc

```diff
@@ -125,7 +112,7 @@ The landing page can route users to a specific dashboard, application, or saved
 
 To configure the landing page, use the default route setting in
 <<kibana-general-settings, Stack Management > {kib} > Advanced settings>>.
-For example, you might set the default route to `/app/kibana#/dashboards`.
+For example, you might set the default route to `/app/dashboards`.
```

Comment:
This example hasn't been valid for a while 😅  Updated the screenshot to match too.

Reply:
Thank you.